src/hooks/mocha.ts

```diff
@@ -0,0 +1,66 @@
+import { simple as walk } from "acorn-walk";
+import type { ESTree } from "meriyah";
+import { expressionFor } from ".";
+import { call_, identifier } from "../generate";
+import { EventEmitter } from "node:stream";
+import Recording from "../Recording";
+import { recording, start } from "../recorder";
+
+export function shouldInstrument(url: URL): boolean {
+  return url.pathname.endsWith("/mocha/lib/runner.js");
+}
+
+export function transform(program: ESTree.Program): ESTree.Program {
+  return patchRuntime(program);
```

Comment:
This `patchRuntime` function body should just go into the `transform` body, especially seeing the name isn't actually accurate anymore (it was referring to jest's runtime module).

Reply:
Done